kubehub/models/cloud_provider.py

```diff
@@ -1,16 +1,21 @@
 from django.db import models
+import subprocess
 
 
 class CloudProvider(models.Model):
+    name_max = int(subprocess.check_output("getconf NAME_MAX /", shell=True))
```

Comment:
should not rely on getconf on particular os, it's better to set fixed max for all OS (take minimum of all possible)

Reply:
using the value from limits.h i tried to avoid hardcode, and  since we will use some sort of kernel based image for docker is it necessary to use hard coded value ? coz it will not depend on the user's operating system

Reply:
ok